addons/mail/static/src/components/discuss/discuss_tests.js

```diff
@@ -421,6 +421,205 @@ QUnit.test('sidebar: add channel', async function (assert) {
     );
 });
 
+QUnit.test('sidebar: pinned channel 1: init with one pinned channel', async function (assert) {
```

Comment:
This file is too big, we need to split it badly.

I would suggest to start now by creating a new file `discuss_pinned_tests.js` and move those there.